doc/sphinx/changes.rst

```diff
@@ -8,6 +8,687 @@ Recent changes
+  fixes `#13336 <https://github.com/coq/coq/issues/13336>`_,
+  by Hugo Herbelin).
+- **Fixed:**
+  The case of tactics, such as :tacn:`eapply`, producing existential variables
+  under binders with an ill-formed instance
+  (`#13373 <https://github.com/coq/coq/pull/13373>`_,
+  fixes `#13363 <https://github.com/coq/coq/issues/13363>`_,
+  by Hugo Herbelin).
+
+Tactic language
+^^^^^^^^^^^^^^^
+
+- **Added:**
+  An if-then-else syntax to Ltac2
+  (`#13232 <https://github.com/coq/coq/pull/13232>`_,
+  fixes `#10110 <https://github.com/coq/coq/issues/10110>`_,
+  by Pierre-Marie Pédrot).
+- **Fixed:**
+  Printing of the quotation qualifiers when printing :g:`Ltac` functions
+  (`#13028 <https://github.com/coq/coq/pull/13028>`_,
+  fixes `#9716 <https://github.com/coq/coq/issues/9716>`_
+  and `#13004 <https://github.com/coq/coq/issues/13004>`_,
+  by Hugo Herbelin).
+
+SSReflect
+^^^^^^^^^
+
+- **Added:**
+  SSReflect intro pattern ltac views ``/[dup]``, ``/[swap]`` and ``/[apply]``
+  (`#13317 <https://github.com/coq/coq/pull/13317>`_,
+  by Cyril Cohen).
+- **Fixed:**
+  Working around a bug of interaction between + and /(ltac:(...)) cf
+  `#13458 <https://github.com/coq/coq/issues/13458>`_
+  (`#13459 <https://github.com/coq/coq/pull/13459>`_,
+  by Cyril Cohen).
+
+Commands and options
+^^^^^^^^^^^^^^^^^^^^
+
+- **Changed:**
+  When compiled with OCaml >= 4.10.0, Coq will use the new best-fit GC
+  policy, which should provide some performance benefits. Coq's policy
+  is optimized for speed, but could increase memory consumption in
+  some cases. You are welcome to tune it using the ``OCAMLRUNPARAM``
+  variable and report back on good settings so we can improve the defaults.
+  (`#13040 <https://github.com/coq/coq/pull/13040>`_,
+  fixes `#11277 <https://github.com/coq/coq/issues/11277>`_,
+  by Emilio Jesus Gallego Arias).
+- **Changed:**
+  Drop prefixes from grammar non-terminal names,
+  e.g. "constr:global" -> "global", "Prim.name" -> "name".
+  Visible in the output of :cmd:`Print Grammar` and :cmd:`Print Custom Grammar`.
+  (`#13096 <https://github.com/coq/coq/pull/13096>`_,
+  by Jim Fehrle).
+- **Changed:**
+  When declaring arbitrary terms as hints, unsolved
+  evars are not abstracted implicitly anymore and instead
+  raise an error
+  (`#13139 <https://github.com/coq/coq/pull/13139>`_,
+  by Pierre-Marie Pédrot).
+- **Changed:**
+  Option `-native-compiler` of the configure script now impacts the
+  default value of the `-native-compiler` option of coqc. The
+  `-native-compiler` option of the configure script is added as an on demand
+  value, which becomes the default, thus preserving the previous default
+  behavior.
+  The stdlib is still precompiled when configuring with `-native-compiler
+  yes`. It is not precompiled otherwise.
+  This an implementation of point 2 of
+  `CEP #48 <https://github.com/coq/ceps/pull/48>`_
+  (`#13352 <https://github.com/coq/coq/pull/13352>`_,
+  by Pierre Roux).
```

Comment:
Same problem. This one should probably be in "Tools".

Also, a wording suggestion:
```suggestion
- **Changed:**
  Option `-native-compiler` of the configure script now impacts the
  default value of the `-native-compiler` option of coqc. The
  `-native-compiler` option of the configure script supports a new `ondemand`
  value, which becomes the default, thus preserving the previous default
  behavior.
  The stdlib is still precompiled when configuring with `-native-compiler
  yes`. It is not precompiled otherwise.
  This an implementation of point 2 of
  `CEP #48 <https://github.com/coq/ceps/pull/48>`_
  (`#13352 <https://github.com/coq/coq/pull/13352>`_,
  by Pierre Roux).
```

Reply:
Done